components/responsive-contentful-visual.vue

```diff
@@ -0,0 +1,135 @@
+<!-- Conditionally render landscape or portrait visual instances -->
+
+<script lang='coffee'>
+import ContentfulVisual, { makeSrcset } from './contentful-visual'
+import { ucFirst } from '../services/helpers'
+export default
+	name: 'ResponsiveContentfulVisual'
+	props: {
+
+		# Merge craft-visual props
+		...ContentfulVisual.props
+
+		# Make explicit landscape and portrait props
+		landscapeImage: Object|Array
+		portraitImage: Object|Array
+		landscapeVideo: Object|Array
+		portraitVideo: Object|Array
+	}
+	data: ->
+		mounted: false
+		isLandscape: null
+	mounted: ->
+		@isLanscapeMediaQuery = window.matchMedia '(orientation: landscape)'
+		@checkIsLandscape @isLanscapeMediaQuery
+		@isLanscapeMediaQuery.addListener @checkIsLandscape
+		@mounted = true
+	destroyed: ->
+		@isLanscapeMediaQuery?.removeListener @checkIsLandscape
+	computed:
+		# Visual configs
+		landscape: -> @makeConfig @getAsset('image', 'landscape'),
+			@getAsset('video', 'landscape')
+		portrait: -> @makeConfig @getAsset('image', 'portrait'),
+			@getAsset('video', 'portrait')
+		# Do we have unique landscape and portrait configs?
+		isResponsive: -> !!(@landscape and @portrait)
```

Comment:
Can you restore the newlines between code blocks like this?

```suggestion
		# Visual configs
		landscape: -> @makeConfig @getAsset('image', 'landscape'),
			@getAsset('video', 'landscape')
		portrait: -> @makeConfig @getAsset('image', 'portrait'),
			@getAsset('video', 'portrait')
			
		# Do we have unique landscape and portrait configs?
		isResponsive: -> !!(@landscape and @portrait)
```

Something about how you copied the source from https://github.com/BKWLD/cloak/blob/master/components/responsive-craft-visual.vue destroyed all those newlines.